gameClientToGameWindowTests/helpers/readGames.ts

```diff
@@ -1,4 +1,26 @@
 import * as fs from 'fs';
+import { config } from '../../config/config';
+import { Game } from '../../types';
 
-export const readGames = () =>
-    JSON.parse(fs.readFileSync('games.json', 'utf-8'));
+export const readGames = () => {
+    const providers = config.getProviders();
+    const games = JSON.parse(fs.readFileSync('games.json', 'utf-8'));
+    const gamesList: Game[] = [];
+
+    if (!providers || !providers.length) {
```

Comment:
It feels like we need to understand how to avoid having to do this better. I don't know if we are importing our env vars correctly or have setup our tests right. I'm going to do some Googling to see how to get Playwright to recognise all the env vars from the start phase